tests/integration/conftest.py

```diff
@@ -67,14 +71,13 @@ class Lwm2mServer(HelperBase):
     """Server subclass of HelperBase"""
 
     def __init__(self, arguments="", timeout=3, encoding="utf8"):
-        base_path = str(Path(__file__).parent.absolute())
-        self.pexpectobj = pexpect.spawn(base_path +
-                                        "/../../build-wakaama/examples/server/lwm2mserver " +
+        self.pexpectobj = pexpect.spawn(REPO_BASE_PATH +
```

Comment:
If `REPO_BASE_PATH` would be an object of Path, this could be more pythonic.

Reply:
Yes. But again, `pexpect` is not pythonoic as it reqires a string as path argument.